buildconfig/BuildConfig.h

```diff
@@ -151,6 +151,9 @@ class Config {
      */
     QString TECHNIC_API_BUILD = "multimc";
 
+    QString MODRINTH_STAGING_URL = "https://staging-api.modrinth.com/v2";
```

Comment:
Might make sense to remove this since it's not used anywhere

Reply:
we can keep it in case we need to quickly switch to the staging api for testing and such i think :)